### ISSUE TITLE
Suggestion: Game Interface Text

### ISSUE THREAD

**Issue by LukkasVN** [kevincerro-dvrv/canon-defense#47]
The game interface seems too small or difficult to notice, I recommend to make it bigger and noticeable by the user.

Also showing the new wave in the middle of the screen may help the player notice the difficulty increase.

**Comment by BraisGlezArias** [comment]
I agree, at first I thought there was no indicator about your ammo or points. It was not until I played in the city background that I was able to see that there was indeed such indicators.
